### PR TITLE
Remove pricing link from header

### PR DESCRIPTION
**Pull request by miiila**
Experiment from Sales department: let's see, if removal of pricing will increase or decrease amount of singups.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <title>{{ page.title }}&mdash;Apiary Blog</title>
 
   <meta name="viewport" content="width=device-width, minimum-scale=1.0">
-  <meta name="author" content="Apiary">
+  <meta name="author" copntent="Apiary">
```
Comment (XVincentX) [inline _layouts/default.html]: 😱

Comment (kylef) [inline _layouts/default.html]: Seems like an introduction of a typo here.

Comment (miiila) [inline _layouts/default.html]: https://github.com/apiaryio/apiaryio.github.com/pull/101/commits

**Comment by JackuB** [comment]
Interesting. Want to keep it in the footer?

**Comment by miiila** [comment]
@JackuB need to ask. Currently creating PR's for app, blog and help.

**Comment by miiila** [comment]
So final decision is (at least currently) to go with /plans page. Please *DO NOT MERGE* until https://github.com/apiaryio/apiary/pull/4074 is merged.

**Pull request merged**